raiden/raiden_event_handler.py

```diff
@@ -543,6 +558,29 @@ def handle_contract_send_channelsettle(
             block_identifier='latest',
         )
 
+    def update_pfs(self, raiden: RaidenService, event: Event):
+        channel_state = get_channelstate_by_token_network_and_partner(
+            chain_state=state_from_raiden(raiden),
+            token_network_id=to_canonical_address(
+                event.balance_proof.token_network_identifier,
+            ),
+            partner_address=to_canonical_address(event.recipient),
+        )
+        if channel_state is not None:
+            msg = UpdatePFS.from_balance_proof(
+                balance_proof=event.balance_proof,
+                reveal_timeout=channel_state.reveal_timeout,
+            )
+            msg.sign(raiden.signer)
+            raiden.transport.send_global(PATH_FINDING_BROADCASTING_ROOM, msg)
```

Comment:
`send_global` can block the greenlet for a while if the room isn't existing (or the server is slow to respond etc.).
 Is that ok here? If not then this should be in it's own greenlet.

Reply:
I interpret the thumbs up as: it's intended to block here, so no change.